lib/pkgcloud/azure-v2/compute/client/servers.js

```diff
@@ -187,15 +197,30 @@ function createServer(options, callback) {
         'properties': {
           'publisher': 'Microsoft.Compute',
           'type': 'CustomScriptExtension',
-          'typeHandlerVersion': '1.4',
+          'typeHandlerVersion': '1.8',
           'settings': {
-            'fileUris': [
-              'https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-vm-winrm-windows/ConfigureWinRM.ps1',
-              'https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-vm-winrm-windows/makecert.exe',
-              'https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-vm-winrm-windows/winrmconf.cmd'
-            ],
-            'commandToExecute': '[concat(\'powershell -ExecutionPolicy Unrestricted -file ConfigureWinRM.ps1 \',variables(\'hostDNSNameScriptArgument\'))]'
-          }
+            'fileUris': ["https://raw.githubusercontent.com/CatalystCode/pkgcloud/master/lib/pkgcloud/azure-v2/scripts/ssh.ps1"],
+            'commandToExecute': `powershell -File ssh.ps1 ${hostname}\\${options.username} ${options.password}`
+          },
+        }
+      }];
+    } else { // linux - make sure the new user is in sudoers - so he can sudo with no password
+      template.resources[vmIndex].resources = [{
+        'type': 'Microsoft.Compute/virtualMachines/extensions',
```

Comment:
It does work, but in none ubuntu linux by default the azure user is not a passwordless sudoer, it has to enter his password before sudoing - so this patch fixes that.